test/parallel/test-stream-readable-readingMore.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+const Readable = require('stream').Readable,
+	assert = require('assert');
+
+const readable = new Readable({
+	read(size) {}
+});
+
+// false initially
+assert.strictEqual(readable._readableState.readingMore, false);
+
+readable.on('data', data => {
```

Comment:
Since there are assertions inside of this callback, can you wrap it in `common.mustCall()`.